org.beagleboard.imagingutility.metainfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this file should be taken upstream post corrections https://github.com/beagleboard/bb-imager-rs/

Reply:
Moved